examples/lsm303_accel_inclinometer.py

```diff
@@ -0,0 +1,38 @@
+""" Display inclination data five times per second """
+
+import time
+from math import atan2, degrees
+import board
+import busio
+import adafruit_lsm303_accel
+
+
+i2c = busio.I2C(board.SCL, board.SDA)
+sensor = adafruit_lsm303_accel.LSM303_Accel(i2c)
+
+
+def vector_2_degrees(x, y):
+    radians = atan2(y, x)
+    degrees_calc = degrees(radians)
+    if degrees_calc < 0:
+        degrees_calc = 360 + degrees_calc
+    return degrees_calc
```

Comment:
Could be shortened to something like
```python
angle = degrees(atan2(y, x))
if angle < 0:
    angle += 360
return angle
```

Reply:
Same here, will make this change tonight.